portfolio/src/main/webapp/index.html

```diff
@@ -8,39 +8,72 @@
   </head>
   <body>
     <div id="content">
+      <div id="menu">
+        <h2>
+        <a href="index.html">Home</a>&nbsp&nbsp
```

Comment:
I'm trying to use the whitespace here to add spaces between text and not for controlling whether the text wraps. white-space: nowrap would not work for that since it collapses spaces, can I get a link to the style guide to see the other options?
https://www.w3schools.com/cssref/pr_text_white-space.asp indicates an option using CSS is to use pre-wrap and the space ' '  character rather than &nbsp. Would that be better? I used &nbsp because then you can easily see where the spaces are.

Reply:
Oh thank for the context! I think its fine to keep it iwht &nbsp.